webapp/src/utils.ts

```diff
@@ -211,6 +211,10 @@ class Utils {
         }
     }
 
+    static assureProtocol(url: string): string {
+        return url.match(/^.{3,9}:\/\//) ? url : `https://${url}`
```

Comment:
Nit:

This currently matches antyhing like `afdsffafasd://` as long as it's between 3-9 characters.
A slight improvement would be to do `^(http|https)://` regex

Reply:
Also I think 
![image](https://user-images.githubusercontent.com/17804942/118828099-f75d0c00-b88a-11eb-9f6f-fa686a01a433.png) 
should always be showing up once you enter a valid URL.

It only shows up for me once after I hover over the link area which is confusing.

Reply:
> This currently matches antyhing like notright:// as long as it's between 3-9 characters before the ://.
A slight improvement would be to do ^(http|https):// regex

I had the same thought but but then I though why to restrict to those protocols. For example, someone can use FTP, magnet, web socket or anything. Thoughts?

Reply:
I had this thought after but what if my schema is like mattermost:// or something. If it's less than 3 or greater than 9 characters, it will display an error. In my opinion, I would say restrict to http/https protocol _for the time being_ and if in the future for whatever reason we need others, we can remove that limitation. 

This also  comes from testing in notion which doesn't accept protocols other than http/https for it's links.

Reply:
Done 👍